jmxtrans-utils/src/main/java/com/googlecode/jmxtrans/util/NumberUtils.java

```diff
@@ -43,4 +43,26 @@ public static boolean isNumeric(Object value) {
 		return false;
 	}
 
+	public static boolean isValidNumber(Object value) {
+		if (!(value instanceof Number)) {
```

Comment:
Although the string representations of NaN and infinity don't currently pass the existing `IsNumeric`